Gulpfile.js

```diff
@@ -270,7 +270,7 @@ gulp.step('test-server-run', () => {
         return gulp
             .src('test/server/*-test.js', { read: false })
             .pipe(mocha({
-                timeout: getTimeout(2000)
+                timeout: getTimeout(4_000)
```

Comment:
Could you please clarify why you increased this value?

Reply:
I increased it only because each `.profile()` test took ~3.5 seconds on my average mac. should I revert it?